great_expectations/expectations/expectation.py

```diff
@@ -1269,6 +1277,18 @@ def run_diagnostics(
         If errors are encountered in the process of running the diagnostics, they are assumed to be due to
         incompleteness of the Expectation's implementation (e.g., declaring a dependency on Metrics
         that do not exist). These errors are added under "errors" key in the report.
+
+        Args:
+            raise_exceptions_for_backends (bool)
+            ignore_suppress (bool)
```

Comment:
@lockettks were these intended to be empty?

Reply:
nope-fixed!